modules/cluster-logging-elasticsearch-exposing.adoc

```diff
@@ -177,6 +177,7 @@ The response appears similar to the following:
     "max_score": 1,
     "hits": [
       {
+        // TBD - Update from `.operations` to `infra-` in the following example for https://bugzilla.redhat.com/show_bug.cgi?id=1873372
```

Comment:
You can use the up2date curl from this PR above to provide a fresh example output.

Reply:
Sorry - I'm working to get a realistic environment set up that can generate such an output. However, I'm not there yet. I'll create an approximation of the output for now.